otcextensions/sdk/vlb/v3/load_balancer.py

```diff
@@ -29,20 +29,37 @@ class LoadBalancer(resource.Resource):
         'description', 'flavor_id', 'name', 'project_id', 'provider',
         'vip_address', 'vip_network_id', 'vip_port_id', 'vip_subnet_id',
         'vip_qos_policy_id', 'provisioning_status', 'operating_status',
-        'availability_zone', is_admin_state_up='admin_state_up',
+        'availability_zone_list', is_admin_state_up='admin_state_up',
+        availability_zone='availability_zone_list',
     )
 
     # Properties
-    #: The administrative state of the load balancer *Type: bool*
-    is_admin_state_up = resource.Body('admin_state_up', type=bool)
     #: Name of the target Octavia availability zone
-    availability_zone = resource.Body('availability_zone', type=list)
+    availability_zone = resource.Body('availability_zone_list', type=list)
```

Comment:
what if we do `availability_zones`. We make it easier not to force list2string conversion where we require to normalize things